parser-typechecker/unison/Main.hs

```diff
@@ -59,6 +67,13 @@ usage = P.callout "🌻" $ P.lines [
         <> "Multiple transcript files may be provided; they are processed in sequence"
         <> "starting from the same codebase.",
   "",
+  P.bold "ucm transcript.fork -preserve mytranscript.md",
+  P.wrap $ "Executes the `mytranscript.md` transcript in a copy of the current codebase"
+        <> "and creates `mytranscript.output.md` if successful. Exits after completion,"
+        <> "and maintains the temporary directory created."  
+        <> "Multiple transcript files may be provided; they are processed in sequence"
+        <> "starting from the same codebase.",
+  "",
```

Comment:
Suggestions on the wording more than welcome. The "-codebase" flag is supported for transcripts and does take "precedence" before "-preserve", should that be mentioned?

Reply:
I was just thinking that `-preserve` makes it sound like we stop short of deleting the resulting codebase (which is true!)  I think I'd rather ignore the fact that the temporary codebase exists at all; as it's just an implementation detail after all.  Suppose that the temporary codebase only existed in memory, unless the user requested to have it saved on disk at the end of the run.  What could we call the flag in that case?  Maybe `-preserve` still makes sense, but I wouldn't be surprised if there was something clearer.  `-save` or `-write` come to mind, just want it to be clear enough that the resulting codebase is the thing that's being saved, as opposed to raising a question about whether the transcript output is saved.

Along the same vein, instead of "maintains the temporary directory created", I'd rather say something like "saves the resulting codebase into a new directory on disk".

Reply:
I agree that ideally we should be hiding the implementation detail, and communicating the end result, which is you end up with a codebase stored in a path. `-save-codebase` or `-store-codebase`? `-preserve` feels closely coupled to the "I am starting by creating a directory and then..." underlying implementation, your point is valid.

Reply:
Yeah, maybe `-save-codebase`?